deploy/ydb-operator/values.yaml

```diff
@@ -57,7 +57,15 @@ webhook:
     #  nodePort: 9443
     #
     ## Arbitrary fqdn for WebhookConfiguration instead of a default Service cluster fqdn:
-    #  fqdn: example.org
+    fqdn: example.org
```

Comment:
Just to double check, are you sure we want `example.org` as a default value or should it be commented out?